Assignments/Object oriented/hw2.js

```diff
@@ -2,11 +2,19 @@ CB.CloudApp.init('ayyuhziequij', 'e592ce9a-b29b-444c-b5b2-138f4572644f');
 
 class Product {
 	constructor(name, price) {
-		let product = new CB.CloudObject("Product");
-		product.set("name",name);
-		product.set("price",price);
+		this.name = name;
+		this.price = price;
 	}
-	save() {
+
+	api() {
+		const product = new CB.CloudObject("Product");
```

Comment:
If you set the `product` to `this.cloudObject` then you will be able to access it throughout the class. Changing it to `cloudObject` will also make it easier to read.